jruby/index.md

```diff
@@ -15,7 +15,7 @@ JRuby is likely the only Ruby JIT that has ever had significant use in productio
 
 The JRuby source code is available at [our archive](https://github.com/ruby-compiler-survey/jruby) or the [original repository](https://github.com/jruby/jruby).
 
-We're discussing JRuby 9.2.9.0, `458ad3ed9cdb`, 30 October 2019, which is the latest release as at 3 February 2020. JRuby by default will compile the entire entry-point script on load. This makes it hard to understand the compilation of individual methods, the effect of profiling, and things like compilation threshold. This feature cannot be disabled with an option, so we turn it off by patching the source code.
+We're discussing JRuby 9.2.9.0, `458ad3ed9cdb`, 30 October 2019, which is the latest release as of 3 February 2020. JRuby by default will compile the entire entry-point script on load. This makes it hard to understand the compilation of individual methods, the effect of profiling, and things like compilation threshold. This feature cannot be disabled with an option, so we turn it off by patching the source code.
```

Comment:
'as at' and 'as of' are both idiomatic British English.

https://english.stackexchange.com/questions/112770/understanding-as-of-as-at-and-as-from

Reply:
Interesting, I never saw it used before.
My personal impression when reading it is it reads really badly, but I'm not a native English speaker.